### PR TITLE
Add development tools team

### PR DESCRIPTION
**Pull request by pecigonzalo**
This still a draft which is 90% a copy of the Distribution role.

- [x] Requires: https://github.com/sourcegraph/about/pull/1555

#### Tasks
- [ ] Define vision and goals
- [ ] Define hiring requirements and interview process

/cc @nicksnyder

### CODE REVIEW

**Comment by nicksnyder** [comment]
the diff here includes some things that aren't what you made

**Comment by pecigonzalo** [comment]
@nicksnyder Its intentional, this requires #1555 which moves the JDs here.

File: handbook/engineering/developer-tools/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Developer tools team
```
Comment (nicksnyder) [inline handbook/engineering/developer-tools/index.md:1]: I think the right way to document this is not to document a new team that doesn't exist, but instead add a goal to the distribution team and document this in the distribution team's growth plan (e.g. we will grow this team and plan on spinning off a dev tools team in the future.

Examples:
https://about.sourcegraph.com/handbook/engineering/web#growth-plan
https://about.sourcegraph.com/handbook/engineering/cloud#growth-plan

**Comment by nicksnyder** [comment]
I merged that PR so you can rebase

File: handbook/engineering/developer-tools/index.md (hunk 1 above)
Comment (pecigonzalo) [reply handbook/engineering/developer-tools/index.md:1]: I can merge this back to distribution, but I was actually using what we did in https://github.com/sourcegraph/about/pull/1150/ for the "Backend infrastructure team" as an example.

**Comment by pecigonzalo** [comment]
This still a draft and Im still changing it.

File: handbook/engineering/developer-tools/index.md (hunk 1 above)
Comment (nicksnyder) [reply handbook/engineering/developer-tools/index.md:1]: Ah, I didn't realize we still had that (we had some other prospective teams that I deleted and merged back into existing teams).

You can leave this if you want, but also update the distribution team to reference the growth plan.

Comment (pecigonzalo) [reply handbook/engineering/developer-tools/index.md:1]: I would like to keep it as an initial idea of that team's vision and goals, but I will add this to distribution's a distribution growth plan so its reflected there.

File: handbook/engineering/developer-tools/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+**The developer tools team does not yet exist** and currently works as part of the [distribution team](../distribution/index.md).
+
+## Vision
+
+Create tools, systems and workflows that empower Sourcegraph engineers to work more efficiently and build quality products.
```
Comment (tsenart) [inline handbook/engineering/developer-tools/index.md:7]: This seems to have overlap with some of the goals the future [Backend Infrastructure team was supposed to have](https://about.sourcegraph.com/handbook/engineering/backend-infrastructure#vision). Is there a way to articulate this to remove this ambiguity? Or is there some actual overlap that we need to remove?

Comment (pecigonzalo) [reply handbook/engineering/developer-tools/index.md:7]: Do you mean the vision or the goals below? I think both teams will likely have similar goals, but target different parts.
As I understand the backend team will focus more on internal libraries, while this team will focus on CI/CD and dev tooling.

Comment (tsenart) [reply handbook/engineering/developer-tools/index.md:7]: I meant the vision.

> I think both teams will likely have similar goals, but target different parts.

Agreed. Can we make it clear what those differences are in the Vision statement?

> As I understand the backend team will focus more on internal libraries

Libraries and shared infrastructure.

Comment (pecigonzalo) [reply handbook/engineering/developer-tools/index.md:7]: > Libraries and shared infrastructure.

What type of shared infrastructure? Do you have something in mind?

> Can we make it clear what those differences are in the Vision statement?

Ill give it a look as I continue to update this document, is there a particular concern or a suggestion you have in mind? I dont think there is that much overlap to cause a problem at the moment.

Comment (tsenart) [reply handbook/engineering/developer-tools/index.md:7]: > What type of shared infrastructure? Do you have something in mind?

Examples: git-server and postgres.

> is there a particular concern or a suggestion you have in mind? I dont think there is that much overlap to cause a problem at the moment.

Just want to make sure we're aligned on these teams vision & long term goals and that its clear what their domains of ownership are. Would a candidate be able to understand the difference by reading the two teams pages?

Comment (pecigonzalo) [reply handbook/engineering/developer-tools/index.md:7]: I believe they would.

**Pull request closed without merging**